### DISCUSSION TITLE
Biblioteca para manipulação de hora e data

### DISCUSSION THREAD

**Discussion by diogogyn** [dgadelha/Portugol-Webstudio#237] [Dúvidas / Perguntas] [answered]
Olá, sou professor de programação e tenho utilizado o portugol para ensino de programação para meus alunos.
Eu estava procurando por uma forma  para manipular hora e data com esta ferramenta. Gostaria de saber se há uma biblioteca não divulgada oficialmente para fazer uso.

**Comment by dgadelha** [comment]
Olá!

Existe uma biblioteca nativa chamada `Calendario`, existe pouca documentação dela, mas você consegue encontrar o código-fonte dela aqui: https://github.com/dgadelha/Portugol-Webstudio/blob/main/packages/runtime/src/libs/Calendario.ts

**Reply by dgadelha** [reply, marked as answer]
Fechando pois o recurso de documentação de bibliotecas foi lançado.
